### PR TITLE
Fix issues with internal API requestBody in API spec

### PR DESCRIPTION
**Pull request by CodyWMitchell**
## What?
Some of the information in our `openapi.yaml` and `openapi.json` were incorrect.

## Why?
Allow other applications to clearly understand how to integrate with the export-service API

## How?
- Updated the requestBody of the `upload` endpoint to expect `application/json` or `application/csv`
- Updated the requestBody of the `error` endpoint to expect the correct query parameters and `application/json`
- Gave an example of the `json` sent to the `error` endpoint

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [X] General Coding Practices

### CODE REVIEW

File: static/spec/openapi.yaml [hunk 1]
```diff
@@ -189,14 +193,13 @@ paths:
...
-          in: query
-          schema:
-            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              example: { "message": "not found", "error": 404 }
```
Comment (dehort) [inline static/spec/openapi.yaml:202]: Should we be more specific with what the error messages should look like?  Should we make the `message` and `error` fields required?

**Review by dehort — APPROVED** [review]
Looks good to me!

File: static/spec/openapi.yaml (hunk 1 above)
Comment (CodyWMitchell) [reply static/spec/openapi.yaml]: Updated to
```
"requestBody": {
          "required": true,
          "content": {
            "application/json": {
              "schema": {
                "type": "object",
                "required": [
                  "message",
                  "error"
                ],
                "properties": {
                  "message": {
                    "type": "string",
                    "description": "A human-readable error message describing the problem",
                    "example": "The requested resource does not exist"
                  },
                  "error": {
                    "type": "integer",
                    "description": "The HTTP status code of the error",
                    "minimum": 400,
                    "maximum": 599,
                    "example": 404
                  }
```

**Review by dehort — APPROVED** [review]
Looks good to me!

**Pull request merged**